src/client/editor/prefabs.js

```diff
@@ -506,17 +514,70 @@ PrefabCollider.prototype.setRadius = function(radius) {
 	}
 };
 
+PrefabCollider.prototype.setModel = function(modelName) {
+	if(this.colliderData.shape !== "mesh") return;
+
+	// Remove from references
+	if(this.colliderData.model) {
+		delete modelsTab.models[this.colliderData.model].prefabEntities[this.uuid];
+		this.sceneObject.geometry = new THREE.Geometry();
```

Comment:
Any specific reason for using THREE.Geometry over `THREE.BufferGeometry`? As per the documentation:
>Geometry is a user-friendly alternative to BufferGeometry. Geometries store attributes (vertex positions, faces, colors, etc.) using objects like Vector3 or Color that are easier to read and edit, but less efficient than typed arrays.
>
> Prefer BufferGeometry for large or serious projects.

https://threejs.org/docs/#api/en/core/Geometry

Reply:
In this case it's empty placeholder geometry, since setting it to null causes rendering to fail. It's replaced by `THREE.BufferGeometry` in other cases. (Except for mesh colliders actually, so that'll need a change!)